### PR TITLE
#352 Automate the setting up of FILE_ID into pg_backup.config

### PR DESCRIPTION
**Pull request by VladimirRyzhkov**
For #352 : Automate the setting up of FILE_ID into pg_backup.config, we already have the FILE_ID in the script, so what's left is to change it in file

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#439](https://github.com/timelesslounge/timelessis/issues/439) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#439](https://github.com/timelesslounge/timelessis/issues/439) is assigned to @vladarefiev[/z](https://www.0crat.com/u/vladarefiev), here is [why](https://www.0crat.com/footprint/CF9LG1YCA/31e8aee0-8b7c-4580-9688-647d42e0bf6b); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @emilianodellacasa[/z](https://www.0crat.com/u/emilianodellacasa) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

File: scripts/backup/pg_backup.sh [hunk 1]
```diff
@@ -107,8 +107,7 @@ DB_FILE_GREP=$(gdrive list | grep $FILENAME)
 if [[ $? -ne 0 ]]; then
     echo -e "[ERROR]["$(date +\%Y-\%m-\%d\ %H:%M:%S:%3N)"] Could not recover the FILE_ID ourselves, try running \"gdrive list\" and recover the ID for file \"$FILENAME\" to set pg_backup.config's FILE_ID"
 else
-    echo "Set your pg_backup.config's FILE_ID to this ID:"
+    DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
     FILE_ID="$(echo $DB_FILE_GREP | head -1 | cut -d" " -f1)"
-    echo $FILE_ID
+    sed -i -e 's/^FILE_ID=.*/FILE_ID='"$FILE_ID"'/g' $DIR/pg_backup.config
```
Comment (vladarefiev) [inline scripts/backup/pg_backup.sh]: @VladimirRyzhkov 
`^FILE_ID=.*` we catch every thing after `FILE_ID` to end line. is it ok?

**Comment by vladarefiev** [comment]
@VladimirRyzhkov I know that it's not your choice, but I'm surprised, why should we use `bash` if we have great script language as `python`

File: scripts/backup/pg_backup.sh (hunk 1 above)
Comment (VladimirRyzhkov) [reply scripts/backup/pg_backup.sh]: @vladarefiev Yes, sed cares by himself about it. For example, I appended `FOO_ATTR=456` after `FILE_ID=` and then run to set value `foo-123`:
```
# Google Drive backup settings
SERVICE_ACCOUNT_CREDENTIALS=timeless-db-backup.json
TIMEOUT=0
BACKUP_PATH=./
FILE_ID=foo-123
FOO_ATTR=456

######################################

```

**Comment by vladarefiev** [comment]
@rultor merge

**Comment by emilianodellacasa** [comment]
@rultor merge

**Comment by 0crat** [comment]
Job was finished in 24 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))

**Comment by 0crat** [comment]
@ypshenychka[/z](https://www.0crat.com/u/ypshenychka) please review this job completed by @vladarefiev[/z](https://www.0crat.com/u/vladarefiev), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
Job [#439](https://github.com/timelesslounge/timelessis/issues/439) is not in the agenda of @vladarefiev[/z](https://www.0crat.com/u/vladarefiev), can't inspect

**Comment by 0crat** [comment]
The job [#439](https://github.com/timelesslounge/timelessis/issues/439) is now out of scope

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @emilianodellacasa[/z](https://www.0crat.com/u/emilianodellacasa)

**Comment by ypshenychka** [comment]
@vladarefiev According to our [QA Rules](http://www.zerocracy.com/policy.html#42):
>The code reviewer found at least three problems in the code.
Comments were mostly about design problems, not cosmetic issues.

No issues were found during code review. 
Please confirm that you'll try to find at least three major problems while future reviews.

**Comment by vladarefiev** [comment]
@ypshenychka Confirm

**Comment by ypshenychka** [comment]
@vladarefiev thanks

**Comment by ypshenychka** [comment]
@0crat quality bad

**Comment by 0crat** [comment]
Quality is low, no payment, see [§31](http://www.zerocracy.com/policy.html#31)

**Comment by 0crat** [comment]
Quality review completed: +8 point(s) just awarded to @ypshenychka[/z](https://www.0crat.com/u/ypshenychka)

**Pull request merged**